pontoon/api/schema.py

```diff
@@ -112,6 +115,54 @@ def resolve_localizations(obj, _info, include_disabled):
         return qs.filter(project__disabled=False)
 
 
+class ActionLog(DjangoObjectType):
+    class Meta:
+        model = ActionLogModel
+        only_fields = (
+            "action_type",
+            "created_at",
+            "performed_by",
+            "translation",
+        )
+
+
+class User(DjangoObjectType):
+    class Meta:
+        convert_choices_to_enum = False
+        model = UserModel
+        only_fields = (
+            "username",
+            "first_name",
+            "last_name",
```

Comment:
It's just so that we don't expose user ids in the API. I'll remove `last_name`, and then we have `first_name` for users who have filled it, and `username` as a backup. Note that I intend this API to be mostly used to generate dashboards, which is why it matters to be able to expose users.

+1 on email addresses, though. That will make this a lot more useful.